src/Http/Controllers/OrganizationController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Http\Requests\StoreOrganizationRequest;
+use Blumilk\Meetup\Core\Http\Requests\UpdateOrganizationRequest;
+use Blumilk\Meetup\Core\Models\Organization;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\View\View;
+
+class OrganizationController extends Controller
+{
+    public function index(): View
+    {
+        $organizations = Organization::orderBy("name")->paginate(20);
```

Comment:
```suggestion
        $organizations = Organization::query()->orderBy("name")->paginate(20);
```

With static `query()` method used as first in the chain you're getting full support of query builder in PHPStorm. :)